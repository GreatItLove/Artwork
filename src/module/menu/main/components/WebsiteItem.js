import React from 'react';
import { ListItem } from 'material-ui/List';
import PropTypes from 'prop-types';

import { WEBSITES_PATH } from '../../../../config/routesPatch';

const propTypes = {
  style: PropTypes.object.isRequired,
  styleChild: PropTypes.object.isRequired
};

/**
 * @function WebsiteItem
 * @param {Object} style
 * @param {Object} styleChild
 * @returns {XML}
 */
function WebsiteItem({ style, styleChild }) {
  return (
    <ListItem
      key={WEBSITES_PATH}
      primaryText="Website"
      innerDivStyle={style}
      leftIcon={<i className="material-icons">web</i>}
      primaryTogglesNestedList={true}
      nestedItems={[
        <ListItem
          key="ContentManagement"
          innerDivStyle={styleChild}
          primaryText="Content Management"
          primaryTogglesNestedList={true}
          nestedItems={[
            <ListItem key="About" innerDivStyle={styleChild} primaryText="About" />,
            <ListItem key="ArtistsMain" innerDivStyle={styleChild} primaryText="Artists Main" />,
            <ListItem key="ArtistsDetail" innerDivStyle={styleChild} primaryText="Artists Detail" />,
            <ListItem key="ArtistsInfo" innerDivStyle={styleChild} primaryText="Artists Info" />,
            <ListItem key="ArtworkDetail" innerDivStyle={styleChild} primaryText="Artwork Detail" />,
            <ListItem key="ArtworkShop" innerDivStyle={styleChild} primaryText="Artwork Shop" />,
            <ListItem key="ArtworkInquiry" innerDivStyle={styleChild} primaryText="Artwork Inquiry" />,
            <ListItem key="ContactUs" innerDivStyle={styleChild} primaryText="Contact Us" />,
            <ListItem key="CopyrightPrivacy" innerDivStyle={styleChild} primaryText="Copyright/Privacy" />,
            <ListItem key="Ecommerce" innerDivStyle={styleChild} primaryText="Ecommerce" />,
            <ListItem key="Events" innerDivStyle={styleChild} primaryText="Events" />,
            <ListItem key="Exhibits" innerDivStyle={styleChild} primaryText="Exhibits" />,
            <ListItem key="FAQPage" innerDivStyle={styleChild} primaryText="FAQ Page" />,
            <ListItem key="HomePage" innerDivStyle={styleChild} primaryText="Home Page" />,
            <ListItem key="MapPage" innerDivStyle={styleChild} primaryText="Map Page" />,
            <ListItem key="MyCollectionPage" innerDivStyle={styleChild} primaryText="My Collection Page" />,
            <ListItem key="NewsPage" innerDivStyle={styleChild} primaryText="News Page" />,
            <ListItem key="PrivateViewingPage" innerDivStyle={styleChild} primaryText="Private Viewing Page" />,
            <ListItem key="ResourcesPage" innerDivStyle={styleChild} primaryText="Resources Page" />,
            <ListItem key="SearchPage" innerDivStyle={styleChild} primaryText="Search Page" />,
            <ListItem key="ServicesPage" innerDivStyle={styleChild} primaryText="Services Page" />,
            <ListItem key="SlideShow" innerDivStyle={styleChild} primaryText="Slide Show" />,
            <ListItem key="Submissions Page" innerDivStyle={styleChild} primaryText="Submissions Page" />,
            <ListItem key="SubscribePage" innerDivStyle={styleChild} primaryText="Subscribe Page" />
          ]}
        />,
        <ListItem
          key="DesignCenter"
          innerDivStyle={styleChild}
          primaryText="Design Center"
          primaryTogglesNestedList={true}
          nestedItems={[
            <ListItem key="Overview" innerDivStyle={styleChild} primaryText="Overview" />,
            <ListItem key="NavigationLinks" innerDivStyle={styleChild} primaryText="Navigation Links" />,
            <ListItem key="StyleFilesCSS" innerDivStyle={styleChild} primaryText="Style Files (CSS)" />,
            <ListItem key="DesignBasics" innerDivStyle={styleChild} primaryText="Design Basics" />
          ]}
        />
      ]}
    />
  );
}

WebsiteItem.propTypes = propTypes;

export default WebsiteItem;
