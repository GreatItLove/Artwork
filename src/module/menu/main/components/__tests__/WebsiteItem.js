import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import WebsiteItem from '../WebsiteItem';

describe('WebsiteItem', () => {
  it('Snapshot of WebsiteItem', () => {
    const wrapper = shallow(<WebsiteItem style={{}} styleChild={{}} />);

    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
