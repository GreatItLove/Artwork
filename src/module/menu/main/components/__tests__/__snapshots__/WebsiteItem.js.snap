// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WebsiteItem Snapshot of WebsiteItem 1`] = `
<ListItem
  autoGenerateNestedIndicator={true}
  containerElement="span"
  disableKeyboardFocus={false}
  disabled={false}
  initiallyOpen={false}
  innerDivStyle={Object {}}
  insetChildren={false}
  leftIcon={
    <i
      className="material-icons"
    >
      web
    </i>
  }
  nestedItems={
    Array [
      <ListItem
        autoGenerateNestedIndicator={true}
        containerElement="span"
        disableKeyboardFocus={false}
        disabled={false}
        initiallyOpen={false}
        innerDivStyle={Object {}}
        insetChildren={false}
        nestedItems={
            Array [
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="About"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Artists Main"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Artists Detail"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Artists Info"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Artwork Detail"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Artwork Shop"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Artwork Inquiry"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Contact Us"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Copyright/Privacy"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Ecommerce"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Events"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Exhibits"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="FAQ Page"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Home Page"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Map Page"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="My Collection Page"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="News Page"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Private Viewing Page"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Resources Page"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Search Page"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Services Page"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Slide Show"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Submissions Page"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Subscribe Page"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
              ]
        }
        nestedLevel={0}
        onKeyboardFocus={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        onNestedListToggle={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}
        open={null}
        primaryText="Content Management"
        primaryTogglesNestedList={true}
        secondaryTextLines={1}
    />,
      <ListItem
        autoGenerateNestedIndicator={true}
        containerElement="span"
        disableKeyboardFocus={false}
        disabled={false}
        initiallyOpen={false}
        innerDivStyle={Object {}}
        insetChildren={false}
        nestedItems={
            Array [
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Overview"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Navigation Links"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Style Files (CSS)"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
                <ListItem
                  autoGenerateNestedIndicator={true}
                  containerElement="span"
                  disableKeyboardFocus={false}
                  disabled={false}
                  initiallyOpen={false}
                  innerDivStyle={Object {}}
                  insetChildren={false}
                  nestedItems={Array []}
                  nestedLevel={0}
                  onKeyboardFocus={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onNestedListToggle={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  open={null}
                  primaryText="Design Basics"
                  primaryTogglesNestedList={false}
                  secondaryTextLines={1}
            />,
              ]
        }
        nestedLevel={0}
        onKeyboardFocus={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        onNestedListToggle={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}
        open={null}
        primaryText="Design Center"
        primaryTogglesNestedList={true}
        secondaryTextLines={1}
    />,
    ]
  }
  nestedLevel={0}
  onKeyboardFocus={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  onNestedListToggle={[Function]}
  onTouchEnd={[Function]}
  onTouchStart={[Function]}
  open={null}
  primaryText="Website"
  primaryTogglesNestedList={true}
  secondaryTextLines={1}
/>
`;
